.gitignore

```diff
@@ -1 +1,5 @@
 vendor/
+*.servicemonitor
+*.configmap
+*.prometheusrules
+*-serviceAccount
```

Comment:
What are these files? I don't think we generate them anyway, so that we need to ignore them, do we?

Reply:
Maybe it was wrong to put them here. So the thing is, they're generated interim files in our build process, sometimes `build.sh` doesn't remove them in my local setup, I just wanted to put them here to prevent accidental commits. Better to remove them.